Updates to Container Apps Docs (#17038)
DataDog/documentation: content/en/serverless/azure_container_apps/_index.md

### 3. Results

> Once the deployment is completed, your metrics and traces are sent to Datadog. In Datadog, navigate to the [APM Trace Explorer] and search for your Container App by service name, or filter using the facet `origin:containerapp` to see your Azure Container App traces.

Reviewer: ```suggestion
Once the deployment is completed, your metrics and traces are sent to Datadog. In Datadog, navigate to the [APM Trace Explorer][11] and search for your Container App by service name, or filter using the facet `origin:containerapp` to see your Azure Container App traces.
```

## Additional configurations

- **Advanced Tracing:** The Datadog Agent already provides some basic tracing for popular frameworks. Follow the [advanced tracing guide][2] for more information.

> - **Logs:** You can set the `DD_LOGS_ENABLED` environment variable to `true` to capture application logs and dispatch them to Datadog through the serverless instrumentation directly. To send logs without direct instrumentation, you can set up central [log collection via the Azure Integration][1], then use the [Diagnostic Settings on the Container App Environment][] to direct the logs to the configured logs pipeline. 

Reviewer: ```suggestion
- **Logs:** You can set the `DD_LOGS_ENABLED` environment variable to `true` to capture application logs and send them to Datadog through the serverless instrumentation directly. To send logs without direct instrumentation, set up central [log collection through the Azure Integration][1], then use the [Diagnostic Settings on the Container App Environment][10] to direct the logs to the configured logs pipeline. 
```

[7]: https://learn.microsoft.com/en-us/azure/container-apps/manage-secrets
[8]: /metrics/distributions/
[9]: /metrics/#time-and-space-aggregation

> [10]: https://learn.microsoft.com/en-us/azure/container-apps/log-options#diagnostic-settings

Reviewer: ```suggestion
[10]: https://learn.microsoft.com/en-us/azure/container-apps/log-options#diagnostic-settings
[11]: https://app.datadoghq.com/apm/traces
```
